src/Serokell/Util/Version.hs

```diff
@@ -0,0 +1,43 @@
+module Serokell.Util.Version
+       ( -- * Git revision
+         retrieveGitRev
+       ) where
+
+import Universum
+
+import Instances.TH.Lift ()
+import System.Environment (lookupEnv)
+import System.Exit (ExitCode (..))
+import System.Process (readProcessWithExitCode)
+
+import qualified Data.Text as Text
+import qualified Language.Haskell.TH as TH
+import qualified Language.Haskell.TH.Syntax as TH
+
+{- | Gets the git revision.
+For example, you can use it configuring CLI options:
+
+@
+versionOption = infoOption
+    ("Git revision: " <> toString $(retrieveGitRev))
+    (long "version" <> help "Show version")
+@
+
+Note, that you'll need @-XTemplateHaskell@.
```

Comment:
Could you also add note that in order to see latest git revision for latest commit you should force recompilation of file which contains `$(retrieveGitRev)`?